libs/wingsdk/src/target-sim/react-website.ts

```diff
@@ -0,0 +1,70 @@
+import { Construct } from "constructs";
+import { ISimulatorResource } from "./resource";
+import { ReactWebsiteSchema, REACT_WEBSITE_TYPE } from "./schema-resources";
+import { bindSimulatorResource, makeSimulatorJsClient } from "./util";
+import * as cloud from "../cloud";
+import * as core from "../core";
+import * as ex from "../ex";
+import { IInflightHost } from "../std";
+import { BaseResourceSchema } from "../testing/simulator";
+// import { simulatorAttrToken } from "./tokens";
+
+export class ReactWebsite
+  extends ex.ReactWebsite
+  implements ISimulatorResource
+{
+  constructor(scope: Construct, id: string, props: ex.ReactWebsiteProps) {
+    super(scope, id, props);
+    if (!this._isDevRun) {
+      // In the future we can create an host (proxy like) for the development one if needed
+      this._websiteHost = cloud.Website._newWebsite(
+        this,
+        `${this.node.id}-host`,
+        {
+          ...this._hostProps,
+          path: this._buildPath,
+        }
+      );
+
+      this.node.addDependency(this._websiteHost);
+      core.Connections.of(this).add({
+        source: this,
+        target: this._websiteHost,
+        name: `host`,
+      });
+    }
+  }
+
+  public toSimulator(): BaseResourceSchema {
+    const schema: ReactWebsiteSchema = {
+      type: REACT_WEBSITE_TYPE,
+      path: this.node.path,
+      props: {
+        path: this._projectPath,
+        startCommand: this._startCommand,
+        environmentVariables: Object.fromEntries(
+          this._environmentVariables.entries()
+        ),
+        isDevRun: this._isDevRun,
+        url: this.url,
+      },
+      attrs: {},
+    };
+    return schema;
+  }
+
+  public bind(host: IInflightHost, ops: string[]): void {
+    bindSimulatorResource(__filename, this, host);
+    super.bind(host, ops);
+  }
+
+  /** @internal */
+  public _toInflight(): string {
+    return makeSimulatorJsClient(__filename, this);
+  }
+
+  /** @internal */
+  public _getInflightOps(): string[] {
+    return [];
+  }
```

Comment:
I think we might be able to move this to the base class since `ReactWebsite` has the same inflight methods on all targets